README.md

```diff
@@ -5,21 +5,22 @@ in the `cEP-0000.md` document.
+| [cEP-0010](cEP-0010.md) | Convert bears to use aspects       | This document describes how `aspects` will be defined for and connected with bears and how their results will be annotated with them.                                                                                                                                                                                                        |
+| [cEP-0012](cEP-0012.md) | coala's Command Line Interface     | This cEP describes the design of coala's Command Line Interface (the action selection screen).                                                                                                                                                                                                                                               |
+| [cEP-0013](cEP-0013.md) | Cobot Enhancement and porting      | This cEP describes about the new features that are to be added to cobot as a part of the [GSoC project](https://summerofcode.withgoogle.com/projects/#4913450777051136).                                                                                                                                                                     |
+| [cEP-0014](cEP-0014.md) | Generate relevant coafile sections | This cEP proposes a framework for coala-quickstart to generate more relevant `.coafile` sections based on project files and user preferences.                                                                                                                                                                                                |
+| [cEP-0018](cEP-0018.md) | Integration of ANTLR into coala    | This cEP describes how an API based on ANTLR will be constructed and maintained                                                                                                                                                                                                                                                              |
+| [cEP-0019](cEP-0019.md) | Meta-review System                 | This cEP describes the details of the process of implementing the meta-review system as a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5188493739819008).                                                                                                                                                     |
+| [cEP-0020](cEP-0020.md) | Newcomer Gamification system       | This cEP describes a gamification system to automate the recognition of activities performed by people who join the coala community, and encourage them to progress towards obtaining the role of "Developer" in the community. This cEP is a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5892040252981248). |
```

Comment:
You have made 400 characters and changed whole table. Can you make your description short?

Reply:
> Can you make your description short?

Yes, I can, I can make it something like:
```
This cEP describes the details of the process of implementing the newcomer gamification system as a part of the GSoC'18 project.
```
But I am not sure as that would take away the feel of the whole sentence :smile: 

Let @jayvdb, decide that, as he is also my mentor ;)

Reply:
Maybe consider something like:

This cEP describes a gamification system to help in recognition of activities and to encourage members who want to join our community obtain the role of "Developer". This cEP is a part of the [GSoC'18 project](https://summerofcode.withgoogle.com/projects/#5892040252981248).

Reply:
agree, find a shorter description